mc-sema/cfgToLLVM/x86Instrs_Misc.cpp

```diff
@@ -709,6 +709,141 @@ static InstTransResult translate_LEA64_32r(NativeModulePtr natM, BasicBlock *&bl
     return doLeaRef<32>(natM, block, ip, inst);
 }
 
+static InstTransResult translate_CPUID32(NativeModulePtr natM, BasicBlock *&block, InstPtr ip, MCInst &inst) {
```

Comment:
What do you think about having `__mcsema_cpuid(RegState *)` inside of the ELF64 and ELF32 drivers? Then other things like `RDTSC` and be similarly handled.